docs/proposal-link-definition-and-disambiguation.md

```diff
@@ -0,0 +1,51 @@
+# Proposal: Link Definition and Disambiguation
+
+## Current status
+
+- We generate link ref defs from wiki-links inline
+- Wiki links are expected to be the full source of truth, but we don't have enough information to conclusively disambiguate
```

Comment:
Slapping comments here just by reading this from top to bottom, at this point got a question "disabiguate what"?

Reply:
Pushed a disambiguation to this disambiguation :)